src/firefly/js/visualize/saga/ImageMetaDataWatcher.js

```diff
@@ -0,0 +1,179 @@
+/*
+ * License information at https://github.com/Caltech-IPAC/firefly/blob/master/License.txt
+ */
+
+import {take} from 'redux-saga/effects';
+import {union,get,isEmpty} from 'lodash';
+import {TABLE_NEW,TABLE_SELECT,TABLE_HIGHLIGHT,
+        TABLE_REMOVE,TABLE_UPDATE} from '../../tables/TablesCntlr.js';
+import ImagePlotCntlr, {visRoot, dispatchPlotImage, dispatchChangeActivePlotView} from '../ImagePlotCntlr.js';
+import {REINIT_RESULT_VIEW} from '../../core/AppDataCntlr.js';
+import {findTblById,gatherTableState} from '../../tables/TableUtil.js';
+import {primePlot} from '../PlotViewUtil.js';
+import MultiViewCntlr, {dispatchReplaceImages, getViewerPlotIds, 
+                        getMultiViewRoot, getViewer, GRID_FULL, SINGLE} from '../MultiViewCntlr.js';
```

Comment:
I only case I am still using export default for controllers with a large set of action constants. Otherwise I am exporting directly.